src/components/RequestDashboard/SettingsSection.jsx

```diff
@@ -0,0 +1,282 @@
+  };
+
+  const resetSettings = () => {
+    originalValues.forEach(e => {
+      try {
+        updateSetting(e[0], e[1]);
+      } catch {
+        console.log('Failed to reset setting value');
+      }
+    });
+  };
+  const clearQuestionnaireResponses =
+    ({ defaultUser }) =>
+    () => {
+      props.client
+        .request('QuestionnaireResponse?author=' + defaultUser, { flat: true })
+        .then(result => {
+          result.forEach(resource => {
+            props.client
+              .delete('QuestionnaireResponse/' + resource.id)
+              .then(result => {
+                console.log(result);
+              })
+              .catch(e => {
+                console.log('Failed to delete QuestionnaireResponse ' + resource.id);
+                console.log(e);
+              });
+          });
+        })
+        .catch(e => {
+          console.log('Failed to retrieve list of QuestionnaireResponses');
+          console.log(e);
+        });
+    };
+
+  const resetPims =
+    ({ pimsUrl }) =>
+    () => {
+      let url = new URL(pimsUrl);
+      const resetUrl = url.origin + '/doctorOrders/api/deleteAll';
+      console.log('reset pims: ' + resetUrl);
+
+      fetch(resetUrl, {
+        method: 'DELETE'
+      })
+        .then(response => {
+          console.log('Reset pims: ');
+          console.log(response);
+        })
+        .catch(error => {
+          console.log('Reset pims error: ');
+          console.log(error);
+        });
+    };
+  const resetRemsAdmin =
+    ({ cdsUrl }) =>
+    () => {
+      let url = new URL(cdsUrl);
+      const resetUrl = url.origin + '/etasu/reset';
+
+      fetch(resetUrl, {
+        method: 'POST'
+      })
+        .then(response => {
+          console.log('Reset rems admin etasu: ');
+          console.log(response);
+        })
+        .catch(error => {
+          console.log('Reset rems admin error: ');
+          console.log(error);
+        });
+    };
+  const clearMedicationDispenses =
+    ({ ehrUrl, access_token }) =>
+    () => {
+      console.log('Clear MedicationDispenses from the EHR: ' + ehrUrl);
+      const client = FHIR.client({
+        serverUrl: ehrUrl,
+        ...(access_token ? { tokenResponse: access_token } : {})
+      });
+      client
+        .request('MedicationDispense', { flat: true })
+        .then(result => {
+          console.log(result);
+          result.forEach(resource => {
+            console.log(resource.id);
+            client
+              .delete('MedicationDispense/' + resource.id)
+              .then(result => {
+                console.log(result);
+              })
+              .catch(e => {
+                console.log('Failed to delete MedicationDispense ' + resource.id);
+                console.log(e);
+              });
+          });
+        })
+        .catch(e => {
+          console.log('Failed to retrieve list of MedicationDispense');
+          console.log(e);
+        });
+    };
+  const reconnectEhr =
```

Comment:
since this uses the same arguments as RequestBuilder.js's reconnectEhr, maybe this file's and RequestBuilder's reconnectEhr should be separated into a function into some sort of utility file?

Reply:
the request builder's version is not connected to any button, so it has been removed.